tallyarbiter-m5stickc.ino

```diff
@@ -245,9 +250,11 @@ void processTallyData() {
     if (getBusTypeById(JSON.stringify(DeviceStates[i]["busId"])) == "\"program\"") {
       if (DeviceStates[i]["sources"].length() > 0) {
         mode_program = true;
+        digitalWrite(led_program, LOW);
```

Comment:
These should be moved down into the evaluateMode() function, otherwise the LED will not light up if the user changes screens and goes back to the Tally screen; leaving it here would only have it light up when new tally data is received

Reply:
Should be fixed in latest push